configs/William/mapped_queue.cpp

```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <deque>
+#include <sys/mman.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <cstring>
+
+
+
+int main(int argc, char** argv){
+
+    char data[64];
+    int num_bytes = 64;
+    char* to_write = (char*)"as"; // needs to be at least 1 byte of data in before calling mmap
+    
+    char* mapped_area; //could replace char with data type/tuple?
```

Comment:
This shouldn't be a `char*``since this is not characters.

It could be a `uint8_t*` if you want to just have "data."